src/tsickle.ts

```diff
@@ -187,6 +188,14 @@ class ClosureRewriter extends Rewriter {
         newDoc.push({tagName: 'abstract'});
       }
 
+      // Add any @template tags.
+      // This will break assuming multiple declarations with conflicting
+      // template variable names.
```

Comment:
I didn't really get it either. It turns out it's not a problem (I think). I was wondering what happens if you have multiple overloads that use the same template arg name (eg. `T`) in different locations and meanings. But I think it's fine because we just make a huge union type mess, so it should all work out.